schema-engine/cli/build.rs

```diff
@@ -2,6 +2,16 @@ use std::process::Command;
 
 fn store_git_commit_hash() {
     let output = Command::new("git").args(["rev-parse", "HEAD"]).output().unwrap();
+
+    // Sanity check on the output.
+    if !output.status.success() {
+        panic!(
+            "Failed to get git commit hash.\nstderr: \n{}\nstdout {}\n",
+            String::from_utf8(output.stderr).unwrap_or_default(),
+            String::from_utf8(output.stdout).unwrap_or_default(),
+        );
+    }
+
     let git_hash = String::from_utf8(output.stdout).unwrap();
     println!("cargo:rustc-env=GIT_HASH={git_hash}");
 }
```

Comment:
nit: would be nicer to have this defined once in some kind of build utils crate that these three crates will depend on as a build dependency

Reply:
Yes, the whole `store_git_commit_hash` could be abstracted

Reply:
Nit postponed to https://github.com/prisma/team-orm/issues/1261